apps/authentication/backends/saml2/views.py

```diff
@@ -136,7 +136,7 @@ def get_advanced_settings():
                 "en": {
                     "name": "JumpServer",
                     "displayname": "JumpServer",
-                    "url": "https://jumpserver.org/"
+                    "url": "https://jumpserver.com/"
                 }
             },
         }
```

Comment:
There is no change needed here; everything looks fine, and there are no known irregularities, potential issues, or optimization suggestions found. The code provided is correct as it stands.